### ISSUE TITLE
Installation error?

### ISSUE THREAD

**Issue by stolken** [makinacorpus/django-leaflet#240]
I followed the short tutorial to install django-leaflet:
`pip install django-leaflet`
Added leaflet to my INSTALLED_APPS in the file `/opt/bitnami/apps/django/django_projects/Project/Project/settings.py`

The map won't show because the leaflet files cannot be found:
```
GET http://35.192.82.255/Project/static/leaflet/leaflet.extras.js net::ERR_ABORTED 404 (Not Found)
35.192.82.255/:7 GET http://35.192.82.255/Project/static/leaflet/leaflet.js net::ERR_ABORTED 404 (Not Found)
(index):15 GET http://35.192.82.255/Project/static/leaflet/leaflet.css net::ERR_ABORTED 404 (Not Found)
(index):11 GET http://35.192.82.255/Project/static/leaflet/leaflet.extras.js net::ERR_ABORTED 404 (Not Found)
(index):12 Uncaught ReferenceError: L is not defined
    at (index):12
(anonymous) @ (index):12
(index):15 GET http://35.192.82.255/Project/static/leaflet/leaflet.css net::ERR_ABORTED 404 (Not Found)
(index):28 Uncaught ReferenceError: L is not defined
    at loadmap ((index):28)
```

I do not see a 'static' directory in /opt/bitnami/apps/django/django_projects/Project/Project/

Am I supposed to create it?

**Comment by anou** [comment]
Any updates on this? I face the same problem... Thx

**Comment by Gagaro** [comment]
Are your other statics available? Is your static files configuration correctly set (https://docs.djangoproject.com/en/3.0/howto/static-files/)?

**Comment by anou** [comment]
Yes all my other statics are available. It is on the staging server that I see in the console : 
<img width="868" alt="Capture d’écran 2020-05-25 à 02 43 18" src="https://user-images.githubusercontent.com/1792534/82893633-99d10080-9f51-11ea-9f9b-8cb02e9921c5.png">

it is searching for some js file here : `app/static/leaflet` 

Fixed with this in Dockerfile :
`&& ln -sf /usr/local/lib/python3.7/site-packages/leaflet/static/leaflet /usr/src/app/static/leaflet`

**Comment by Gagaro** [comment]
Did you do a collectstatic after adding django-leaflet?

**Comment by anou** [comment]
Nope. Nothing like this. Worked like a charm with the basic on local environment but did go likely on staging.

**Comment by renkh** [comment]
You gotta run `python manage.py collectstatic` to copy all the static files into your STATIC_ROOT directory that you specified in settings.py. And then configure your web server to serve the files in STATIC_ROOT. Read more about deploying static files https://docs.djangoproject.com/en/3.0/howto/static-files/deployment/#serving-static-files-from-a-dedicated-server.
